packages/account/src/Sections/Profile/PhoneVerification/phone-verification-card.tsx

```diff
@@ -1,14 +1,34 @@
+import { Tag } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv-com/translations';
```

Comment:
import { useTranslations } from '@deriv-com/translations';

I believe we should import `Localize` from `useTranslations`

Reply:
@matin-deriv I'm using the `Localize` component not the function `localize` from `deriv-com/translation` 🙏